conans/client/installer.py

```diff
@@ -30,7 +31,9 @@ def _init_package_info(deps_graph, paths, current_path):
             conan_file.cpp_info = CppInfo(package_folder)
         else:
             conan_file.cpp_info = CppInfo(current_path)
+
         conan_file.env_info = EnvInfo()
+        conan_file.user_info = UserInfo()
```

Comment:
I don't know why but it's not propagated if I init the object in the conanfile, so, to keep it similar to others I will do it this way.